### PR TITLE
Fixes variant switching in preview mode

### PR DESCRIPTION
**Pull request by Crease29**
- Removes unnecessary div in form
- Adds preview parameter as hidden input because switching variants in preview mode is broken

### CODE REVIEW

**Comment by Sieg** [comment]
Hi @Crease29, Could you please link the related bug, or write more information to reproduce the Issue? It looks working fine.

File: tpl/page/details/inc/productmain.tpl [hunk 1]
```diff
@@ -19,17 +19,18 @@
...
+        [{$oViewConf->getHiddenSid()}]
+        [{$oViewConf->getNavFormParams()}]
+        <input type="hidden" name="cl" value="[{$oView->getClassName()}]">
+        <input type="hidden" name="oxwparent" value="[{$oViewConf->getTopActiveClassName()}]">
+        <input type="hidden" name="listtype" value="[{$oView->getListType()}]">
+        <input type="hidden" name="nocookie" value="1">
+        <input type="hidden" name="cnid" value="[{$oView->getCategoryId()}]">
+        <input type="hidden" name="anid" value="[{if !$oDetailsProduct->oxarticles__oxparentid->value}][{$oDetailsProduct->oxarticles__oxid->value}][{else}][{$oDetailsProduct->oxarticles__oxparentid->value}][{/if}]">
+        <input type="hidden" name="actcontrol" value="[{$oViewConf->getTopActiveClassName()}]">
+        [{if $oConf->getRequestParameter('preview')}]
```
Comment (Sieg) [inline tpl/page/details/inc/productmain.tpl]: There is no $oConf available here :/

**Comment by Crease29** [comment]
Thank you for your review @Sieg .

Steps to reproduce:
1. Login to the administration interface
2. Open an inactive master article that has variants with the preview link at the bottom of the screen
3. Select a variant in the frontend. There should appear an error without my PR.

**Comment by Sieg** [comment]
Hm, yes, I have tried it, works well for me on EE. Will check CE again, anything you can share from EXCEPTION log? Anything custom in shop?

**Comment by Crease29** [comment]
No modules, nothing in the EXCEPTION_LOG. The error for me was, that it just says that the article can't be found, which is obvious because the `preview` parameter was missing in the request that loads the selected variant and as the article is **inactive** it shall not appear in the frontend.

**Comment by Crease29** [comment]
Here you can see it in the demo shop: https://www.youtube.com/watch?v=x__B_phDuag

**Comment by Sieg** [comment]
Thanks! That "active" checkbox was the question :) Have merged your PR recently. Have a nice day!

**Pull request closed without merging**